src/configure/helper/templateHelper.ts

```diff
@@ -310,7 +338,9 @@ let azurePipelineTemplates: { [key in SupportedLanguage]: PipelineTemplate[] } =
                     "type": TemplateAssetType.AzureARMPublishProfileServiceConnection
                 }
             ],
-            azureConnectionType: AzureConnectionType.AzureRMPublishProfile
+            azureConnectionType: AzureConnectionType.AzureRMPublishProfile,
+            templateWeight: 123,
```

Comment:
can you assign same weight for all localTemplates say -> 100. It will be easier in future if we need to change that.

Reply:
umm.. let's decide the weights for the local templates, and I'll update all the templates at once?

Reply:
Have updated the weights to 100